### DISCUSSION TITLE
How can I use this library directly in normal javascript and offline no CDN , no Internet

### DISCUSSION THREAD

**Discussion by Harvindar994** [fingerprintjs/fingerprintjs#963] [Q&A]
I want to use this library in my application that will be running on a local machine and it will not have any Internet connection so can you tell me how can I use it offline?

**Comment by Finesse** [comment]
FingerprintJS (not Fingerprint Pro) is fully offline. [Install it via NPM or Yarn](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/api.md#webpackrollupnpmyarn) and use normally.
